integration-tests/deployment/ccip/token_info.go

```diff
@@ -32,6 +33,7 @@ func (tc *TokenConfig) UpsertTokenInfo(
 func (tc *TokenConfig) GetTokenInfo(
 	lggr logger.Logger,
 	linkToken *burn_mint_erc677.BurnMintERC677,
+	wethToken *weth9.WETH9,
```

Comment:
I'm not super familiar with this code but is this `TokenConfig` struct really needed? We should use the existing offchain config types directly where possible to reduce this churn of code that wraps other stuff IMO. The more "glue code" there is for the tests the less likely it is devs would be effectively working on them. That's an issue with the integration-tests code today as well